src/__tests__/components.test.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import path from 'path'
+import pluginTester from 'babel-plugin-tester'
+import plugin from '../'
+
+const filename = path.resolve(__dirname, '..', '__fixtures__', 'messages.js')
+
+const defaultTest = {
+  title: 'default',
+  code: `
+import { FormattedMessage } from 'react-intl'
+
+console.log(<FormattedMessage defaultMessage="hello" />);
+`,
+}
+
+const multiUseTest = {
+  title: 'multiple uses',
+  code: `
+import { FormattedMessage } from 'react-intl'
+
+console.log(<FormattedMessage defaultMessage="hello" />);
+console.log(<FormattedMessage defaultMessage="another" />);
+`,
+}
+
+const withValueInMessageTest = {
+  title: 'with a value interpolated in the message',
+  code: `
+import { FormattedMessage } from 'react-intl'
+
+console.log(<FormattedMessage defaultMessage={\`hello world ${1}\`} />);
```

Comment:
clear intention `1` →`1 + 1`

Reply:
Hi I'll change that here and in the other test too